src/surfaces/drawerDeprecated/drawerDeprecated.jsx

```diff
@@ -488,7 +491,7 @@ class DrawerDeprecated extends React.Component {
         };
 
         for (t in transitions) {
-            if (el.style[t] !== undefined) {
+            if (el?.style[t] !== undefined) {
```

Comment:
If `el` is indeed nullish, wouldn't `el?style` also be nullish (`undefined`) and thus `el?.style[t]` still be unsafe?

Reply:
would this be safe: `if (el?.style && el.style[t] !== undefined)`?

Reply:
Yeah, that's safer.

Reply:
Looks good.  Resolving.